### ISSUE TITLE
Implement stackframes on events

### ISSUE THREAD

**Issue by natduca** [google/trace-viewer#752]
That is:
https://docs.google.com/document/d/11z5VYKup_PWI8-eBDhm0IDsToOpQ-0pFx8VhZwi21Mc/edit#

**Comment by natduca** [comment]
I believe this to now be working. Closing.

**Comment by dvyukov** [comment]
I've tested this functionality in Go Tracer (using rev 895aa74558d19d91906fb720df6458244ef160c6).
It works like a charm.
I've checked sizes of generated traces:
previous code with stringized stacks: 130MB
no stacks at all: 62MB
with this change, using sf: 64.3M
So it pretty much solves this part of the problem

2 comments:
1. It shows stacks reversed. The frame referenced from sf is shown at the bottom, while the parent is shown at the top. That's not how gdb, lldb, MSVC, Go, breakpad and AddressSanitizer show traces. Is it intentional or merely a mistake?
2. I've tested sf on E, X and I, but I would also like to see it on at least s (stack for start of an arrow). Probably b, e, t need it for consistency as well.

Thanks!
